static/app/views/dashboards/datasetConfig/transactions.spec.tsx

```diff
@@ -0,0 +1,55 @@
+import {OrganizationFixture} from 'sentry-fixture/organization';
+import {PageFiltersFixture} from 'sentry-fixture/pageFilters';
+import {WidgetFixture} from 'sentry-fixture/widget';
+
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {TransactionsConfig} from 'sentry/views/dashboards/datasetConfig/transactions';
+
+describe('TransactionsConfig', function () {
+  describe('getEventsRequest', function () {
+    let api, organization, mockEventsRequest;
+
+    beforeEach(function () {
+      MockApiClient.clearMockResponses();
+
+      api = new MockApiClient();
+      organization = OrganizationFixture();
+
+      mockEventsRequest = MockApiClient.addMockResponse({
+        url: '/organizations/org-slug/events/',
+        body: {
+          data: [],
+        },
+      });
+    });
+
+    it('makes a request to the errors dataset', function () {
```

Comment:
```suggestion
    it('makes a request to the transactions dataset', function () {
```
or `metricsEnhanced` here